### PR TITLE
Configure Cache-Control and Expires headers on the uploaded files.

### PR DESCRIPTION
**Pull request by YoranBrondsema**
This PR adds the ability to configure the `Cache-Control` and `Expires` headers through the `cacheControl` and `expires` configuration options.

Solves https://github.com/ember-cli-deploy/ember-cli-deploy-s3/issues/52

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
     "core-object": "^1.1.0",
     "ember-cli-babel": "^5.0.0",
     "ember-cli-deploy-plugin": "^0.2.2",
-    "lodash": "^3.9.3",
+    "lodash": "^4.11.1",
```
Comment (ghedamat) [inline package.json]: @YoranBrondsema why are we updating this?

I ask because lodash has been giving us quite a bit of gas lately in different plugins and I've seen some weird npm incompatibilities around it

**Comment by ghedamat** [comment]
@YoranBrondsema LGTM

just that comment on lodash /cc @lukemelia

File: package.json (hunk 1 above)
Comment (YoranBrondsema) [inline package.json]: @ghedamat No particular reason but to keep up-to-date. I wasn't aware of any recent problems with lodash, I can revert it back to 3 no problems.

Comment (ghedamat) [inline package.json]: seems stupid but yeah let's revert and maybe do this in another pr
so we can release a beta and get someone to test it just in case.

Comment (YoranBrondsema) [inline package.json]: I understand, no problem.

**Comment by YoranBrondsema** [comment]
@ghedamat Any news on when this can be merged? We'd like to start using this functionality ourselves.

**Comment by ghedamat** [comment]
done :) thanks! will try to cut a release later today

a patch bump should be enough

**Comment by YoranBrondsema** [comment]
@ghedamat Perfect thanks!
